File: code/functions/assemblyscript/ping_pong/ping_pong.ts
import { Event } from "sdk/event";

export function ping(e: Event): u32 {
  let httpEvent = e.http();
  if (httpEvent == null) {
    return 1;
  }

  let resp = httpEvent.unwrap().write("PONG");
  if (resp.err) {
    return 1;
  }

  return 0;
}

File: code/functions/Rust/ping_pong/ping_pong.rs
extern crate taubyte_rs_sdk;
use taubyte_rs_sdk::event::events::Event;




#[no_mangle]
pub unsafe  fn ping(event:Event) -> u32 {
    let http = event.http().unwrap();
    _ = http.write("pong".as_bytes()).unwrap();
    
    return  0
}
